### ISSUE TITLE
[gardena] Bridge doesn't work

### ISSUE THREAD

**Issue by KenkiCK** [openhab/openhab-addons#14038]
After upgrade OH from 3.3 to 3.4 Gardena Binding can't connect the husqvarna Cloud. For the automover binding there is a hint after upgrade to recreate a new cloud thing. I do the same for the gardena thing but this doesn't solve the problem.. Alternative I install the automover binding and enter the credentials (key and secret) here and it works. For Gardena things it's not possible to use it. For me it's actual not a prio 1 topic because it's wintertime.

Herre now the messages of the openhab.log file:

`2022-12-22 10:47:00.396 [INFO ] [ab.event.ThingStatusInfoChangedEvent] - Thing 'gardena:account:c126d5e4bc' changed from INITIALIZING to OFFLINE (COMMUNICATION_ERROR): Warten bis 10:53:33 für einen automatischen Wiederverbindungsversuch

2022-12-22 10:47:11.503 [INFO ] [ab.event.ThingStatusInfoChangedEvent] - Thing 'gardena:account:c126d5e4bc' changed from OFFLINE (COMMUNICATION_ERROR): Warten bis 10:53:33 für einen automatischen Wiederverbindungsversuch to UNINITIALIZED`

**Comment by andrewfg** [comment]
@KenkiCK can you please post a trace log?

**Comment by KenkiCK** [comment]
@andrewfg after I removed the Gardena binding and add it again - it looks like that it works now - the account thing is now online and there are no communication error anymore. 

Before I removed only the bridge (like describe in the install remarks for the Automower Binding) and that doesn't help. It could help for other user if add a hint in the install remarks. 

many thanks and best regards
Christian
